### ISSUE TITLE
[Bug]: Sidebar & thumbnail keyboard navigation focus/tab order is confusing - Accessibility issue

### ISSUE THREAD

**Issue by dmer** [mozilla/pdf.js#19158]
### Attach (recommended) or Link to PDF file

Applies to any PDF viewed in pdfjs
https://mozilla.github.io/pdf.js/web/viewer.html

### Web browser and its version

Chrome Version 131.0.6778.86

### Operating system and its version

Mac OS 14.6.1

### PDF.js version

v4.3.136

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

1. Using the Keyboard, navigate to the "toggle sidebar" button and activate
2. Use the keyboard to enter the list of thumbnails and select a page.

### What is the expected behavior?

When the menu opens, the ideal scenario would be to send focus to the first item in the menu. 

Otherwise, the menu and the thumbnails underneath it should be next in tab order directly after the triggering element in the code.

### What went wrong?

If users open the sidebar menu (first button on the viewer), they need to move backwards to access the controls. To access the thumbnails, they need to tab to the end of the controls along the top of the document and then through all the pages of the PDF and then they are in the thumbnails sidebar. 
The confusing tab order seems to be due to the use of positive tabindex values on the menu items but not the page thumbnails links.

### Additional context

There may be other rules about tab order for keyboard navigation that conflict with this issue, but the current behavior is confusing to users.
This came up in a site Accessibility Audit.

**Comment by Snuffleupagus** [comment]
> ### PDF.js version
> 
> v4.3.136

That version is no longer supported, please find the latest releases at https://mozilla.github.io/pdf.js/getting_started/#download
(Please avoid reporting bugs against old and unsupported versions of the library.)

---

This looks (effectively) like a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1921037

**Comment by dmer** [comment]
@Snuffleupagus 
I did test this against the very latest version here https://mozilla.github.io/pdf.js/web/viewer.html  I specified that using the checkbox in the issue template - was that not the correct form?

This does look like a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1921037 thank you for pointing that out - I did search the Github issues carefully to avoid making a duplicate but I didn't find reference to the other bug list in bugzilla on the github project. I will start searching there as well before I post any further.
